pinot-spi/src/main/java/org/apache/pinot/spi/utils/IngestionConfigUtils.java

```diff
@@ -85,6 +85,34 @@ public static List<AggregationConfig> getAggregationConfigs(TableConfig tableCon
     return null;
   }
 
+  /**
+   * Fetches the configured consistentDataPush boolean from the table config
+   */
+  public static boolean getBatchSegmentIngestionConsistentDataPushEnabled(TableConfig tableConfig) {
+    boolean consistentDataPush = false;
+    if (tableConfig.getIngestionConfig() != null) {
+      BatchIngestionConfig batchIngestionConfig = tableConfig.getIngestionConfig().getBatchIngestionConfig();
+      if (batchIngestionConfig != null) {
+        consistentDataPush = batchIngestionConfig.getConsistentDataPush();
+      }
+    }
+    return consistentDataPush;
+  }
+
+  /**
+   * Fetches the configured disableUniqueSegmentNames boolean from the table config
+   */
+  public static boolean getBatchSegmentIngestionDisableUniqueSegmentNames(TableConfig tableConfig) {
+    boolean disableUniqueSegmentNames = false;
+    if (tableConfig.getIngestionConfig() != null) {
+      BatchIngestionConfig batchIngestionConfig = tableConfig.getIngestionConfig().getBatchIngestionConfig();
+      if (batchIngestionConfig != null) {
+        disableUniqueSegmentNames = batchIngestionConfig.getDisableUniqueSegmentNames();
+      }
+    }
+    return disableUniqueSegmentNames;
+  }
```

Comment:
The functions getBatchSegmentIngestionConsistentDataPushEnabled and getBatchSegmentIngestionDisableUniqueSegmentNames are almost the same, could we fold them into getSegmentIngestionConsistentDataPushConfig() with enum argument that can be either DisableUniqueSegmentNames or ConsistentDataPush. I envision we will need more config like these to support backfill in append tables

Reply:
Yeah, that's a good point and should be considered when we move back to supporting disabling of consistent push. For now, `disableUniqueSegmentNames` field as well as related logic have been reverted a few commits back. 
See https://github.com/yuanbenson/pinot/pull/13#discussion_r946008258